src/components/DiaryCard/DiaryCard.js

```diff
@@ -0,0 +1,45 @@
+import Card from '@material-ui/core/Card';
+import {Button, CardActions, CardContent, Typography} from "@material-ui/core";
+import classes from './DiaryCard.module.css'
+import {useState} from "react";
+
+function DiaryCard(props) {
+    const [more, setMore] = useState(false);
+    const [showButton,setShowButton]=useState('Show More');
```

Comment:
No need of these 3 state hooks `showButton, lessButton, buttonText`. Achieve this functionality without those 3 hooks.

Reply:
I have fixed the issue